static/src/javascripts/projects/commercial/modules/header-bidding/prebid/bid-config.ts

```diff
@@ -161,6 +163,26 @@ const getImprovePlacementId = (sizes: HeaderBiddingSize[]): number => {
 	return -1;
 };
 
+const getImproveSkinPlacementId = (): number => {
+	if (isInUk()) {
+		switch (getBreakpointKey()) {
+			case 'D': // Desktop
+				return 22526482;
```

Comment:
I wonder if its worth documenting where these numbers come from? Or is it obvious given the function name suggests it returns placement ids?

Reply:
I think having a signature and an explicit function name is enough. There is no other place than here where these are defined, to my knowledge.